Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -868,5 +867,11 @@ public void MarkAsDirty()
         {
             LastChanged = DateTime.UtcNow;
         }
+
+        public bool CheckSystemValidity()
+        {
+            //TODO: add other conditions in the following tasks
+            return IsActiveAccordingToDateFields;
```

Comment:
There is already another condition and that is the "Active" flag... that was the purpose of this ... to combine the different reasons as to why it might be valid, Active override stille exists in this branch